src/pipeline/README.md

```diff
@@ -11,6 +11,23 @@ The image processing pipeline is the main construct of our project. When provide
 </p>
 <p align="center">Example of image mosaicing</p>
 
+## Running
+To run the default pipeline you'll need Python 3.10. First, install the necessary requirements with
+
+`pip install -r requirements.txt`
+
+You can then run the default pipeline with
+
+`py main.py`
```

Comment:
This command does not work for me on Mac. Is this a type error?

Reply:
My Python alias is `py`, use `python`; I'll fix this